### PR TITLE
Updating repo list

### PR DESCRIPTION
**Pull request by ryjones**
Signed-off-by: Ry Jones <ry@linux.com>

### CODE REVIEW

**Comment by ryjones** [comment]
@tkuhrt please examine line 181-186 in particular.

File: common/repositories.sh [hunk 1]
```diff
@@ -178,7 +188,7 @@ all_repositories=(
   "${caliper_repositories[@]}"
   "${ursa_repositories[@]}"
   "${grid_repositories[@]}"
-  "${labs_repositories[@]}"
+  "${lab_repositories[@]}"
```
Comment (tkuhrt) [inline common/repositories.sh]: this should be labs_repositories to match line 127.

File: common/repositories.sh [hunk 2]
```diff
@@ -115,39 +119,45 @@ ursa_repositories=(
...
+  https://gerrit.hyperledger.org/r/ci-management
   https://github.com/hyperledger/education.git
   https://github.com/hyperledger/education-cryptomoji.git
+  https://github.com/hyperledger/education-sawtooth-simple-supply.git
   https://github.com/hyperledger/hyperledger.git
   https://github.com/hyperledger/hyperledger-rocket-chat-hubot.git
   https://github.com/hyperledger/hyperledger.github.io.git
   https://github.com/hyperledger/hyperledgerwp.git
+  https://github.com/hyperledger/perf-and-scale-wg.git
+  https://github.com/hyperledger/slack-archive.git
```
Comment (tkuhrt) [inline common/repositories.sh]: I wouldn't want to line_count or get_contributors on this repo.

File: common/repositories.sh [hunk 3]
```diff
@@ -32,6 +32,7 @@ sawtooth_repositories=(
   https://github.com/hyperledger/sawtooth-next-directory.git
   https://github.com/hyperledger/sawtooth-pbft.git
   https://github.com/hyperledger/sawtooth-poet.git
+  https://github.com/hyperledger/sawtooth-private-utxo.git
```
Comment (tkuhrt) [inline common/repositories.sh]: This says ARCHIVED. Thus I removed it.

File: common/repositories.sh [hunk 4]
```diff
@@ -45,17 +46,20 @@ sawtooth_repositories=(
...
 
 iroha_repositories=(
-  https://github.com/hyperledger/iroha.git
   https://github.com/hyperledger/iroha-android.git
   https://github.com/hyperledger/iroha-dotnet.git
   https://github.com/hyperledger/iroha-ed25519.git
   https://github.com/hyperledger/iroha-ios.git
+  https://github.com/hyperledger/iroha-java.git
   https://github.com/hyperledger/iroha-javascript.git
+  https://github.com/hyperledger/iroha-network-tools.git
```
Comment (tkuhrt) [inline common/repositories.sh]: This says ARCHIVED

File: common/repositories.sh (hunk 3 above)
Comment (ryjones) [reply common/repositories.sh]: I transferred it to the archive and updated this line

File: common/repositories.sh (hunk 4 above)
Comment (ryjones) [reply common/repositories.sh]: I transferred it to the archive and updated this line

**Pull request merged**